src/Domain.jl

```diff
@@ -243,7 +243,9 @@ over species.
 # Returns
 Leftover space ∈ [0, 1]
 """
-function relative_leftover_space(loc_coral_cover::Matrix{Float64})::Matrix{Float64}
+function relative_leftover_space(
+    loc_coral_cover::Union{Matrix{Float64},Vector{Float32}}
```

Comment:
One other comment: the way it is now, one can pass a `Vector{Float32}` but this function returning type is `Vector{Float64}`. Won't this cause an error?

Reply:
I'm just testing and ran into an error with `test_rs_w_fig` which seems to be about the clustering. So I'm just running updates and then I'll test but you're right. Should I implement as Pedro suggests though?